src/solvers/LevenbergMarquardt.jl

```diff
@@ -271,9 +281,9 @@ function step_solver!(
     get_residuals!(dmp, lms.candidate_residual_values, temp_x)
 
     ρk =
-        2 * (normFk2 - norm(lms.candidate_residual_values)^2) / (
+        (normFk2 - norm(lms.candidate_residual_values)^2) / (
             -2 * inner(M, lms.p, lms.X, lms.step_vector) - norm(lms.jacF * sk)^2 -
-            λk * norm(sk)
+            λk * norm(sk)^2
```

Comment:
Is there possibly a `^2` missing here?
![image](https://github.com/JuliaManifolds/Manopt.jl/assets/6612981/b1cc46b4-3a27-41fb-bd36-74a9efee1c5c)

Reply:
Yes, I missed that `^2` too, sorry.